command/agent/agent_endpoint_test.go

```diff
@@ -188,19 +189,8 @@ func TestHTTP_AgentSetServers(t *testing.T) {
 		if n := len(servers); n != len(expected) {
 			t.Fatalf("expected %d servers, got: %d: %v", len(expected), n, servers)
 		}
-		received := make(map[string]bool, len(servers))
-		for _, server := range servers {
-			received[server] = true
-		}
-		foundCount := 0
-		for k, _ := range received {
-			_, found := expected[k]
-			if found {
-				foundCount++
-			}
-		}
-		if foundCount != len(expected) {
-			t.Fatalf("bad servers result")
+		if !reflect.DeepEqual(servers, expected) {
```

Comment:
I think you can use assert.Equal here, as it looks like the library is using DeepEquals: https://github.com/stretchr/testify/blob/master/assert/assertions.go#L53